l3gacyb3ta.md

```diff
@@ -0,0 +1,35 @@
+---
+name: "@l3gacyb3ta"
+project: "Collapse-resistant Mesh Networking"
+---
+
+# Project Name
+
+## Summary
+
+I want to build a super simple mesh-network, for a theoretical post-collapse world. I mean collapse as in major infrastructure failing to provide support anymore, so what most people would call off-grid (i guess i have CollapseOS brianrot :3 [^1]). Ideally this would support many forms of interaction, so it could work even if people lost acess to phones. Some previous work I am inspired by is:
+- [meshtastic](https://meshtastic.org/)
+- [disaster.radio](https://disaster.radio/)
+- [LOW←TECH MAGAZINE's solar website](https://solar.lowtechmagazine.com/about.html)
+
+## Plan
+
+1. Research actual networking solutions.
+1. Get a serial-based version working.
+1. Optimize power usage and online-offline transition
+1. Build a web interface.
+1. Test at a larger scale.
+1. Deployment at my school!
+
+## Budget
+
+| Product                       | Supplier/Link                                                                                                                                                    | Cost    |
+|-------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| LILYGO® TTGO US(915MHz) x 3   | https://store.rokland.com/products/lilygo-ttgo-meshtastic-t-beam-v1-1-esp32-lora-915-mhz-wireless-module-wifi-gps-neo-6m-with-oled-display-for-arduino-q202-l206 | $128.30 |
```

Comment:
Would you be needing to solder headers on this thing?

Reply:
Yes! I have the tools for that though